.circleci/config.yml

```diff
@@ -22,20 +22,28 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
+            pip install --user --upgrade pip neuromation
             pip install --user -r .circleci/requirements.txt
       - run:
           name: Build Image
           command: |
             neuro config login-with-token $NEURO_TOKEN
+            neuro config show
             neuro config docker
             export IMAGE_NAME=neuromation/ml-recipe-midi-generator
             export IMAGE_REF=$IMAGE_NAME:$CIRCLE_BUILD_NUM
             export IMAGE_URI=image:$IMAGE_REF
-            make setup \
-                __BAKE_SETUP=yes \
-                CUSTOM_ENV_NAME=$IMAGE_URI \
-                PROJECT_PATH_STORAGE=storage:build-$CIRCLE_BUILD_NUM
-            neuro pull $IMAGE_URI
+            export PROJECT_PATH_STORAGE=storage:build-$CIRCLE_BUILD_NUM
+            export CUSTOM_ENV_NAME=$IMAGE_URI
+            make setup __BAKE_SETUP=yes | tee
+
+            make -f test.mk test_jupyter TRAINING_MACHINE_TYPE=cpu-small | tee
+            make -f test.mk test_jupyter TRAINING_MACHINE_TYPE=gpu-small | tee
+
+            make -f test.mk test_jupyter_baked TRAINING_MACHINE_TYPE=cpu-small | tee
+            make -f test.mk test_jupyter_baked TRAINING_MACHINE_TYPE=gpu-small | tee
```

Comment:
won't this be flaky?

Reply:
you mean because it runs on GPU? on neuro-ai-public (as tests run currently), there are some GPU. Also, cookiecutter template runs tests on GPU when on neuro-ai-public